go-controller/pkg/ovn/egressip.go

```diff
@@ -865,12 +866,21 @@ func (e *EgressIPController) deleteEgressIPAssignments(name string, statusesToRe
 				continue
 			}
 			podNamespace, podName := getPodNamespaceAndNameFromKey(podKey)
-			ni, err := e.networkManager.GetActiveNetworkForNamespace(podNamespace)
-			if err != nil {
-				return fmt.Errorf("failed to get active network for namespace %s", podNamespace)
-			}
 			cachedNetwork := e.getNetworkFromPodAssignment(podKey)
-			err = e.nodeZoneState.DoWithLock(statusToRemove.Node, func(key string) error {
+			var ni util.NetInfo
+			if _, err := e.watchFactory.GetNamespace(podNamespace); err == nil {
+				ni, err = e.networkManager.GetActiveNetworkForNamespace(podNamespace)
```

Comment:
@martinkennelly can we just drop trying to get active network for namespace all together and just rely on the cache?